File: SEP3_UserSystem/src/main/java/mediator/information/ErrorPackage.java
package mediator.information;

public class ErrorPackage extends InformationPackage{
    private boolean IsError;

    public ErrorPackage(String error,boolean isError)
    {
        super(InformationType.ERROR,error);
        IsError = isError;
    }

    public boolean isError() {
        return IsError;
    }
}

File: SEP3_Client/Mediator/Information/ErrorPackage.cs
namespace SEP3_Client.Mediator.Information
{
    public class ErrorPackage : InformationPackage
    {
        public bool Error { get; set; }

        public ErrorPackage(string errorMessage,bool error) : base(InformationType.ERROR,errorMessage)
        {
            Error = error;
        }

        public ErrorPackage() {}

        public bool IsError() {
            return Error;
        }
    }
}